metau-capstone/View Controllers/StudyViewController.m

```diff
@@ -303,7 +305,11 @@ - (CGFloat) progressPercentIncrement {
 - (void) incrementCircleProgress {
     self.percentFinished += [self progressPercentIncrement];
     [self.circleProgressBar setProgress:self.percentFinished animated:YES];
+    PFUser *const user = [PFUser currentUser];
+    user[@"percentFinished"] = @(self.percentFinished);
+    [user saveInBackground];
```

Comment:
I'd encourage you to think about how to avoid storing this if possible. It's susceptible to getting out of sync, failing to save and what not. We could maybe save some calls, similar to the import flow.

There's potentially no need to store it on the backend.